swagger.yaml

```diff
@@ -39,23 +127,160 @@ parameters:
+            description: Default zoom level to use when selecting route on map
+          stops:
+            type: array
+            description: Locations where the bus stops along the route
+            items:
+              properties:
+                stopID:
+                  type: string
+                description:
+                  type: string
+                latitude:
+                  type: number
+                  format: float
+                  description: Latitude of stop
+                longitude:
+                  type: number
+                  format: float
+                  description: Longitude of stop
+                order:
+                  type: integer
+                  description: The order this stop is used relative to the other stops
+  VehiclesResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/VehicleResourceObject"
+  VehicleResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/VehicleResourceObject"
+  VehicleResourceObject:
+    properties:
+      id:
+        type: string
+        description: vehicleID
+      type:
+        type: string
+        description: route
+      attributes:
+        properties:
+          routeID:
+            type: string
+            description: routeID for route of vehicle
+          name:
+            type: string
+            description: Name of vehicle
+          latitude:
+            type: number
+            format: float
+            description: Latitude of vehicle's position
+          longitude:
+            type: number
+            format: float
+            description: Longitude of vehicle's position
+          speed:
+            type: number
+            format: float
+            description: Speed of vehicle (MPH)
+          heading:
+            type: integer
+            description: Compass heading of vehicle (0-359)
+          age:
+            type: integer
+            description: Seconds since vehicle position was updated
+          onRoute:
+            type: boolean
+            description: Whether the vehicle is on its route
+          delayed:
+            type: boolean
+            description: Whether the vehicle is delayed
+  ArrivalsResultObject:
+    properties:
+      data:
+        type: array
+        items:
+          $ref: "#/definitions/ArrivalResourceObject"
+  ArrivalResultObject:
+    properties:
+      data:
+        $ref: "#/definitions/ArrivalResourceObject"
+  ArrivalResourceObject:
+    properties:
+      id:
         type: string
-        description: Git commit hash of the revision which was built.
-      documentation:
+        description: arrivalID
+      type:
         type: string
-        description: Filename of the swagger specification for the API.
+        description: route
+      attributes:
+        properties:
+          routeID:
+            type: string
+            description: routeID for route of vehicle
```

Comment:
i feel like this should be a full URL to the route rather than just the routeID, and maybe it should go in the links section.

same for all the other thingID fields

Reply:
Added a links section but kept these fields since I think it's beneficial to have both.